src/Symfony/Component/Finder/Finder.php

```diff
@@ -721,7 +721,7 @@ private function searchInDirectory(string $dir): \Iterator
         }
 
         if (static::IGNORE_VCS_IGNORED_FILES === (static::IGNORE_VCS_IGNORED_FILES & $this->ignore)) {
-            $gitignoreFilePath = sprintf('%s/.gitignore', $dir);
+            $gitignoreFilePath = $dir.'/.gitignore';
```

Comment:
revert: this change nothing and is the single diff of this file

Reply:
done